jsonnet/openebs-mixin/README.md

```diff
@@ -0,0 +1,62 @@
+# openebs mixin
+
+A set of Grafana dashboards and Prometheus alerts for openebs.
+
+## Generate dashboards and rules
+
+You can manually generate the alerts, dashboards and rules files. 
+
+#### Steps:
+
+1. Install `jsonnet` and `jb`
+	```
+	$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+	$ brew install jsonnet
```

Comment:
How is brew installed?

Reply:
Have to add `sudo apt-get install jsonnet` for linux.
brew is for mac.

Reply:
Add for linux.